policyengine_us/parameters/gov/irs/income/exemption/overtime/weeks_worked.yaml

```diff
@@ -0,0 +1,8 @@
+description: Number of weeks worked in a year.
```

Comment:
remove this file (not a policy parameter) and use the WEEKS_IN_YEAR constant (imported as part of `from policyengine_us.model_api import *`) instead (52)